### ISSUE TITLE
An odd request: is it possible to rename this library?

### ISSUE THREAD

**Issue by Pomax** [PeculiarVentures/Font.js#1]
Hi, 

this is a bit of an odd request, but: myself and others in the typography space have been working on https://github.com/pomax/font.js for the last... oh god almost a decade now, and we just finished a fairly huge overhaul to pull it from ES5, browser only, into ES2020, browser+node format. As part of that overhaul we were going to publish it to npm (it was hosted on the web rather than npm for the last 7 years) and seeing as how you own "fontjs" on the npm registry, we were going to publish it as "font-js" (similar to how my JS Bezier maths library is published as "bezier-js"), but as it turns out npm now forbids publishing packages that are named "similarly" to other packages.

You have zero obligation to do this, of course, but with the long history that font.js has being used in various projects already, would it be possible to rename this library so that Font.js can be published to npm under its own name? If not, that's perfectly understandable, but if so, you'd be helping out the font community a lot (one of Font.js's uses is to underpin https://wakamaifondue.com, which was the primary motivator for starting the uplift rewrite of font.js).

Best regards,

--  Pomax

**Comment by rmhrisk** [comment]
I understand why you might want the package name but we acquired it two years ago for our package that has been filling the void of a browserless font library.

Your library is wonderful, and it is great you intend to make it work in Node now. However since we still have aspirations to do more to ours yet, I don’t feel inclined to give up the package name at this time.

**Comment by Pomax** [comment]
fair enough - the one thing I'd be remiss not to ask is whether you'd consider republishing it as @peculiarventures/fontjs instead, so we can both keep our libraries called Font.js
